libr/cons/cons.c

```diff
@@ -13,6 +11,39 @@ static RCons s_cons_global = {0};
 
 static void __break_signal(int sig);
 
+// Count display lines, treating long lines (>80 chars) as multiple lines
+// Stops counting at R_CONS_MAX_LINES to avoid performance issues with huge outputs
+#define R_CONS_MAX_LINES 10000
```

Comment:
let's use 9000, so we can make the joke and tell the user the lines are over 9000. also remove the R_CONS prefix here because its local to this file. R_CONS must be only the ones in the .h